front/src/components/LoginPanel/comp/RegisterForm.test.js

```diff
@@ -1,20 +1,17 @@
 import { render, screen } from "@testing-library/react";
+import { within } from "@testing-library/dom";
+import userEvent from "@testing-library/user-event";
+import { MockedProvider } from "@apollo/client/testing";
 import RegisterForm from "./RegisterForm";
-import { ApolloProvider } from "@apollo/client/react";
-import { ApolloClient, InMemoryCache } from "@apollo/client";
-
-const client = new ApolloClient({
-    uri: "http://localhost:3001/graphql",
-    cache: new InMemoryCache(),
-});
 
 describe("RegisterForm", () => {
+    var container;
     beforeEach(() => {
-        render(
-            <ApolloProvider client={client}>
+        container = render(
+            <MockedProvider mocks={[]} addTypename={false}>
                 <RegisterForm />
-            </ApolloProvider>
-        );
+            </MockedProvider>
+        ).container;
```

Comment:
You don't use container anywhere, so you don't need return value of render.
It's intended only for special cases, eq. when you really, really must use querySelector, because RTL is not enough.